frontend/packages/topology/src/elements/BaseNode.ts

```diff
@@ -130,7 +140,15 @@ export default class BaseNode<E extends NodeModel = NodeModel, D = any> extends
   }
 
   isGroup(): boolean {
-    return this.group;
+    return this.group && !this.collapsed;
+  }
+
+  isCollapsed(): boolean {
+    return this.collapsed || this.getController().getTypeCollapsed(this.getType());
```

Comment:
This property should be set and read as is regardless of some filter. The controller shouldn't have a collapsed type state.thats dev console specific and related to filters.